examples/python/tridiagonal_solve.py

```diff
@@ -0,0 +1,188 @@
+"""Tridiagonal solve computation HIR generator
```

Comment:
when I try to run this I don't get the correct path out of config. Can you maybe show me either hands on or elaborate what might be wrong?

Reply:
there were a couple of problems with a latest merge with master. Can you try again? 
You just need 
export PYTHONPATH=<wdir>/bundle/install/python

Reply:
(repeated) should this be documented here?